src/seq/index.rs

```diff
@@ -376,6 +380,14 @@ where
 #[cfg(test)]
 mod test {
     use super::*;
+
+    // #[test]
+    // #cfg(feature = "serde1")
+    // fn test_serialization_index_vec() {
+    //     let some_index_vec = IndexVec::from(vec![254,234,2,1]);
+    //     let de_some_index_vec: IndexVec = bincode::deserialize(&bincode::serialize(&some_index_vec).unwrap()).unwrap();
+    // }
```

Comment:
Yes, you can't use `==` on an enum which doesn't support `PartialEq`. You can however deconstruct with `if let IndexVec::U32(v) = de_some_index_vec` or even both simultaneously with `match (some_index_vec, de_some_index_vec) { .. }`.

Bear in mind these aren't exactly equivalent (e.g. one might decide that `U32(u)` and `USize(v)` variants are equal when `u == v`), but in this case we know which variant is in use anyway, so don't need to test the other.